benchmark/sirun/Dockerfile

```diff
@@ -25,10 +26,14 @@ RUN wget -O sirun.tar.gz https://github.com/DataDog/sirun/releases/download/v0.1
 	&& rm sirun.tar.gz \
 	&& mv sirun /usr/bin/sirun
 
+# This list of installed Node.js versions should include all versions used by any active release line
 RUN mkdir -p /usr/local/nvm \
 	&& wget -q -O - https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash \
 	&& . $NVM_DIR/nvm.sh \
-	&& nvm install --no-progress 16.17.1 \
-	&& nvm install --no-progress 18.10.0 \
+	&& nvm install --no-progress 12.22.12 \
+	&& nvm install --no-progress 14.21.3 \
+	&& nvm install --no-progress 16.20.1 \
+	&& nvm install --no-progress 18.16.1 \
+	&& nvm install --no-progress 20.4.0 \
```

Comment:
Is there a reason not to just include only the major so it will install whatever the latest is? I would expect if we're always installing the latest we'd spread out the performance delta versus doing one major update like this. 🤔

Reply:
At any point where we change the version, it then makes it so that all of the historical trend data is then incomparable with the new data. So we should only change versions on very rare and intentional occasions. By not pinning we then get changes at arbitrary times (I believe those times being any time a change to the Dockerfile happens thus invalidating the layer hashes).